adviser/base/argo-workflows/qeb-hwt-integration.yaml

```diff
@@ -0,0 +1,70 @@
+---
+apiVersion: argoproj.io/v1alpha1
+kind: WorkflowTemplate
+metadata:
+  name: thamos-advise
+  annotations:
+    thoth-station.ninja/template-version: 0.2.0
+  labels:
+    app: thoth
+    component: thamos-advise
+spec:
+  templates:
+    - name: thamos-advise
+      inputs:
+        parameters:
+          - name: "GITHUB_EVENT_TYPE"
+          - name: "GITHUB_CHECK_RUN_ID"
+          - name: "GITHUB_INSTALLATION_ID"
+          - name: "GITHUB_BASE_REPO_URL"
+          - name: "GITHUB_HEAD_REPO_URL"
+          - name: "ORIGIN"
+          - name: "REVISION"
+          - name: "THOTH_HOST"
+          - name: "REPO_PATH"
+            value: "/mnt/inputs/artifacts/repository"
+        artifacts:
+          - name: repository
+            path: "{{inputs.parameters.REPO_PATH}}"
+            git:
+              repo: "{{inputs.parameters.GITHUB_HEAD_REPO_URL}}"
+              revision: "{{inputs.parameters.REVISION}}"
+              depth: 1
+      container:
+        name: advise
+        image: workflow-helpers
+        env:
+          - name: WORKFLOW_TASK
+            value: "qeb_hwt"
+          - name: APP_SCRIPT
+            value: "app.sh"
+          - name: GITHUB_EVENT_TYPE
+            value: "{{inputs.parameters.GITHUB_EVENT_TYPE}}"
+          - name: GITHUB_CHECK_RUN_ID
+            value: "{{inputs.parameters.GITHUB_CHECK_RUN_ID}}"
+          - name: GITHUB_INSTALLATION_ID
+            value: "{{inputs.parameters.GITHUB_INSTALLATION_ID}}"
+          - name: GITHUB_BASE_REPO_URL
+            value: "{{inputs.parameters.GITHUB_BASE_REPO_URL}}"
+          - name: GITHUB_HEAD_REPO_URL
+            value: "{{inputs.parameters.GITHUB_HEAD_REPO_URL}}"
+          - name: ORIGIN
+            value: "{{inputs.parameters.ORIGIN}}"
+          - name: THOTH_HOST
+            value: "{{inputs.parameters.THOTH_HOST}}"
+          - name: REPO_PATH
+            value: "{{inputs.parameters.REPO_PATH}}"
+          - name: "WEBHOOK_CALLBACK_URL"
+            value: "http://qeb-hwt-aicoe-prod-bots.cloud.paas.psi.redhat.com"
```

Comment:
can we have a generic value for this? or, we need to put this configuration into a configmap that is created via an overlay. the webhook callback URL seems to be environment specific, therefor the configmap seems to be a good way.

Reply:
but this is a prod value? do we have a test env for the github-webhook receiver? In that case I will create the value in core/base/configmaps.yaml in test? @goern